### PR TITLE
schema: Require that a single note isn’t inside an array

### PR DESCRIPTION
**Pull request by ExE-Boss**
This makes the `notes` field behave the same way as the&nbsp;browser support&nbsp;statement array.

review?(@Elchi3)

### CODE REVIEW

**Comment by Elchi3** [comment]
Do you have thoughts on https://github.com/mdn/browser-compat-data/issues/195 ?

**Comment by ExE-Boss** [comment]
That allowing single&#x2011;element arrays in&nbsp;one context and disallowing them in&nbsp;another is&nbsp;rather inconsistent and we should probably decide one way or&nbsp;the&nbsp;other for&nbsp;both contexts.

**Comment by Elchi3** [comment]
I think we've more or less settled on forbidding single-element arrays, so I think we should accept this PR. It needs a rebase now, though.

**Comment by ExE-Boss** [comment]
review?(@Elchi3): I have now resolved merge conflicts and fixed the build errors (please merge this before more issues arise).

File: test/sample-data.json [hunk 1]
```diff
@@ -208,9 +204,7 @@
                 },
                 {
                   "version_added": "49",
-                  "notes": [
-                    "A single note in an array"
-                  ]
+                  "notes": "A single note in an array"
```
Comment (Elchi3) [inline test/sample-data.json]: This makes no sense now

Comment (ExE-Boss) [reply test/sample-data.json]: That’s because I&nbsp;did a&nbsp;regex replace on&nbsp;the&nbsp;whole repository, and forgot to&nbsp;check for this.

**Pull request merged**